pkg/store/consul/rcstore/fake_store.go

```diff
@@ -62,6 +63,10 @@ func (s *fakeStore) Create(manifest manifest.Manifest, nodeSelector labels.Selec
 	return entry.RC, nil
 }
 
+func (s *fakeStore) CreateTxn(txn *transaction.Tx, manifest manifest.Manifest, nodeSelector labels.Selector, podLabels labels.Set) (fields.RC, error) {
+	panic("transactions not implemented in fake rc store")
```

Comment:
got it. I don't know what to call the functions like `CreateRollingUpdateFromOneMaybeExistingWithLabelSelector`, that's what I was wondering about.